### ISSUE TITLE
False positive in `at-mixin-no-risky-nesting-selector`.

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint-scss/stylelint-scss#1009]
In a list of complex selectors with pseudo classes, within a mixin, and in a nested context, it shouldn't trigger a warning since both parent selectors are at the beginning.
Ex.
```scss
@mixin theme {
  .a {
    &:focus,
    &:hover {
      background-color: #0000;
    }
  }
}
```
